mmdet/datasets/custom.py

```diff
@@ -237,12 +238,13 @@ def get_classes(cls, classes=None):
                 string, take it as a file name. The file contains the name of
                 classes where each line contains one class name. If classes is
                 a tuple or list, override the CLASSES defined by the dataset.
+
+        Returns:
+            tuple[str]: Names of categories of the dataset.
```

Comment:
We may add a conversion from list to tuple before `return` since there are some in list.